### PR TITLE
docs: Add ctrlp/asyncomplete plugin usage

### LINKED ISSUE #9: Add "How to use with ctrlp/asyncomplete.vim" is needed ?
Is there a demand for additions to the docs(vim help)?

* [CtrlP](https://github.com/ctrlpvim/ctrlp.vim/)
* [asyncomplete.vim](https://github.com/prabirshrestha/asyncomplete.vim)

mr.vim support plugins

* [CtrlP source](https://github.com/tsuyoshicho/ctrlp-mr.vim)
* [asyncomplete.vim source](https://github.com/tsuyoshicho/asyncomplete-mr.vim)

### PR DESCRIPTION
**Pull request by tsuyoshicho**
Close #9 
Add ctrlp/asyncomplete usage in help.

Please validate config and review it.

### CODE REVIEW

File: doc/mr.txt [hunk 1]
```diff
@@ -36,6 +38,52 @@ Get |List| of MRU/MRW/MRR by calling |mr#mru#list()|, |mr#mrw#list()|, or
 	:echo mr#mrr#list()
 <
 
+How to use under fuzzy finder or auto completion plugin.
+
+CTRLP USAGE						*mr-usage-ctrlp*
+
+|ctrlp| is Pure Vim script fuzzy finder plugin.
```
Comment (lambdalisue) [inline doc/mr.txt]: Please do NOT refer other plugins that might not be installed on user's system. Instead, please add URL of the plugin.

File: doc/mr.txt [hunk 2]
```diff
@@ -36,6 +38,52 @@ Get |List| of MRU/MRW/MRR by calling |mr#mru#list()|, |mr#mrw#list()|, or
...
+below:
+>
+	" Setting to use MRU, same for MRW and MRR.
+	autocmd User asyncomplete_setup call
+	\ asyncomplete#register_source(
+	\   asyncomplete#sources#mru#get_source_options({
+	\     'name': 'mru',
+	\     'allowlist': ['*'],
+	\     'completor': function('asyncomplete#sources#mru#completor'),
+	\   }))
```
Comment (lambdalisue) [inline doc/mr.txt]: Please follow Vim's default indentation rules. In other word, I prefer the code looks like the below

````vim
	autocmd User asyncomplete_setup call asyncomplete#register_source(
	      \ asyncomplete#sources#mru#get_source_options({
	      \   'name': 'mru',
	      \   'allowlist': ['*'],
	      \   'completor': function('asyncomplete#sources#mru#completor'),
	      \ })
	      \)
```

File: doc/mr.txt [hunk 3]
```diff
@@ -36,6 +38,52 @@ Get |List| of MRU/MRW/MRR by calling |mr#mru#list()|, |mr#mrw#list()|, or
...
+	\     'allowlist': ['*'],
+	\     'completor': function('asyncomplete#sources#anylist#completor'),
+	\     'config': {
+	\       'matcher': '\(\w\|\f\)+$',
+	\       'items': [
+	\         {'name': 'mru', 'function': function('mr#mru#list'), 'args': []},
+	\       ],
+	\     },
+	\   }))
+<
```
Comment (lambdalisue) [inline doc/mr.txt:90]: Same. Please reformat the code.

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: doc/mr.txt (hunk 1 above)
Comment (tsuyoshicho) [reply doc/mr.txt]: Sure, remove content link and add URL link.

File: doc/mr.txt (hunk 2 above)
Comment (tsuyoshicho) [reply doc/mr.txt]: Yes, reformat it.

File: doc/mr.txt (hunk 3 above)
Comment (tsuyoshicho) [reply doc/mr.txt:90]: Ditto.

**Review by lambdalisue — APPROVED** [review]
LGTM. Thanks a lot !

**Pull request merged**